analyzers/src/ITs.JsonParser/IssueParser.cs

```diff
@@ -47,10 +47,17 @@ public static void Execute(string inputPath, string outputPath)
         ConsoleHelper.WriteLineColor("Successfully wrote all Rule reports.", ConsoleColor.Green);
     }
 
-    private static InputReport[] Read(string rootPath) =>
-        Directory.GetFiles(rootPath, "*.json", SearchOption.AllDirectories)
-            .Select(x => new InputReport(x, SerializerOptions))
-            .ToArray();
+    private static InputReport[] Read(string rootPath)
+    {
+        var projects = Directory.GetDirectories(rootPath);
```

Comment:
`projects` doesn't need to be extracted, `foreach(var project n Directory.GetDirectories...` is pretty clear